README.md

```diff
@@ -312,7 +312,15 @@ reference constants. You'll need to explicitly load `app/bot`, then:
 ```ruby
 # config/initializers/bot.rb
 unless Rails.env.production?
-  Dir["#{Rails.root}/app/bot/**/*.rb"].each { |file| require file }
+  api_files = Dir[Rails.root.join('app', 'bot', '**', '*.rb')]
```

Comment:
@elct9620 Probably, we should use something like `bot_files` or `bots` instead or `api_files` in variables?